frontend/src/types.ts

```diff
@@ -450,15 +450,21 @@ export interface FunnelStepRangeEntityFilter extends EntityFilter {
 
 export type EntityFilterTypes = EntityFilter | ActionFilter | FunnelStepRangeEntityFilter | null
 
-export interface PersonType {
-    id?: number
+export interface ActorType {
+    id?: string | number
+    properties: Record<string, any>
+    created_at?: string
+}
+export interface PersonType extends ActorType {
```

Comment:
This is missing the `type` field returned from the API.

However re-purposing the existing type seems like a bad idea, even more since the naming pattern does not match up.